v4/source/system.yaml

```diff
@@ -101,6 +101,53 @@
             // TestEmail
             ok, resp := Client.TestEmail(&config)
 
+  '/site_url/test':
+    post:
+      tags:
+        - system
+      summary: Checks the validity of a Site URL
+      description: |
+        Sends a Ping request to the mattermost server using the specified Site URL.
```

Comment:
Please add minimum server version:
``__Minimum server version__: 5.16``

Reply:
Should we wait until the server PR is merged in order to figure out the version or it's for sure going to be `5.16`?

Reply:
Yup, that makes sense.  Let's wait until it's merged.